### DISCUSSION TITLE
Cuy Animelist semester 4 | sesi 2

### DISCUSSION THREAD

**Discussion by Destaanugra1** [deaafrizal/forum#44] [Troubleshooting Room] [answered]
![Screenshot (124)](https://github.com/deaaprizal/forum/assets/152362800/e7768fee-c71a-4bcf-b368-032b106a1fd0)
bc7e07a0)

![Screenshot (122)](https://github.com/deaaprizal/forum/assets/152362800/3ade49ef-3b1f-4102-9a07-c2802013be1e)

kenapa tidak susai href yg saya buat ya bg atau emang tidak masalah?

**Comment by andikaalakate** [comment, marked as answer]
Coba kirimkan dulu code / screenshot index.js dalam Folder AnimeListnya, bang...

**Reply by Destaanugra1** [reply]
![Screenshot (126)](https://github.com/deaaprizal/forum/assets/152362800/6f3cb3cd-56ae-4d07-b84d-6f59277b8956)
maaf agak telat

**Reply by andikaalakate** [reply]
Link href nya tuh ada typo, /${aime.mal_id} seharusnya => /${anime.mal_id}

**Reply by desta-anugra** [reply]
Makasih banyak bg agak kerepotan emang nyoba" main react

**Reply by andikaalakate** [reply]
Iyaa, sama-sama bro

**Reply by deaafrizal** [reply]
thank u @andikaalakate the best

**Reply by andikaalakate** [reply]
> thank u @andikaalakate the best

Iyaa, sama-sama bang @deaaprizal 👍🏻
